partiql-lang/src/test/kotlin/org/partiql/lang/syntax/PartiQLParserDMLTest.kt

```diff
@@ -0,0 +1,2229 @@
+package org.partiql.lang.syntax
+
+import com.amazon.ion.Decimal
+import com.amazon.ionelement.api.ionDecimal
+import com.amazon.ionelement.api.ionInt
+import com.amazon.ionelement.api.ionString
+import org.junit.Ignore
+import org.junit.Test
+import org.partiql.errors.ErrorCode
+import org.partiql.errors.Property
+import org.partiql.lang.ION
+import org.partiql.lang.domains.id
+import org.partiql.lang.util.getAntlrDisplayString
+import org.partiql.parser.internal.antlr.PartiQLParser
+
+/**
+ * Tests parsing of DML using just the PIG parser.
+ */
+class PartiQLParserDMLTest : PartiQLParserTestBase() {
+    override val targets: Array<ParserTarget> = arrayOf(ParserTarget.DEFAULT)
+
+    // ****************************************
+    // DML
+    // ****************************************
+
+    @Test
```

Comment:
(self-review) DML parsing tests were copied directly from `PartiQLParserTest` and runs on just the PIG parser.